### PR TITLE
Add a back button

### PR DESCRIPTION
**Pull request by breezyboa**
Addresses #48

### CODE REVIEW

**Comment by breezyboa** [comment]
@vipulnsward _a Please review

File: app/javascript/Client/Main.elm [hunk 1]
```diff
@@ -101,24 +108,44 @@ minimizedView =
...
+        showBackButton =
+            case history of
+                Just previousModel ->
+                    case (getSection previousModel.sectionState) of
+                        Blank ->
+                            False
+
+                        Loading ->
+                            False
+
```
Comment (vipulnsward) [inline app/javascript/Client/Main.elm]: Aren't these finite?

Comment (vipulnsward) [reply app/javascript/Client/Main.elm]: I mean the for below line, do we need to catch all by "_"

File: app/javascript/Client/Main.elm [hunk 2]
```diff
@@ -101,24 +108,44 @@ minimizedView =
...
+            (List.concat
+                [ Animation.render model.containerAnimation
+                , [ style
+                        [ ( "position", "fixed" )
+                        , ( "top", "0" )
+                        , ( "background", "#fff" )
+                        , ( "height", "100%" )
+                        , ( "width", "720px" )
+                        , ( "box-shadow", "0 0 50px 0px rgb(153, 153, 153)" )
+                        , ( "font-family", "proxima-nova, Arial, sans-serif" )
```
Comment (vipulnsward) [inline app/javascript/Client/Main.elm]: Hmm, aren't we we moving to styled css yet?

Comment (breezyboa) [reply app/javascript/Client/Main.elm]: @vipulnsward no at least not yet. But I will be moving all such styles to scss files.
The current state of styled elements have certain limitations that are a bit tricky to get around. It makes the code more convoluted. Might as well move them all out to separate files. I will be doing this in a separate PR in which I will also improve the ui

File: app/javascript/Client/Main.elm (hunk 1 above)
Comment (breezyboa) [reply app/javascript/Client/Main.elm]: Did not get you.

**Comment by vipulnsward** [comment]
Please rebase

**Pull request merged**